diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/utils/AstNodeUtils.kt

```diff
@@ -347,7 +347,7 @@ fun ASTNode.findLBrace(): ASTNode? {
         ElementType.WHEN -> this.findChildByType(ElementType.LBRACE)!!
         ElementType.FOR, ElementType.WHILE, ElementType.DO_WHILE ->
             this.findChildByType(ElementType.BODY)?.findChildByType(ElementType.BLOCK)?.findChildByType(ElementType.LBRACE)
-        ElementType.CLASS, ElementType.OBJECT_DECLARATION -> this.findChildByType(ElementType.CLASS_BODY)!!.findChildByType(ElementType.LBRACE)
+        ElementType.CLASS, ElementType.OBJECT_DECLARATION -> this.findChildByType(ElementType.CLASS_BODY)?.findChildByType(ElementType.LBRACE)
         ElementType.FUNCTION_LITERAL -> this.findChildByType(LBRACE)
         else -> this.findChildByType(ElementType.BLOCK)?.findChildByType(ElementType.LBRACE)
```

Comment:
Which cases are currently covered by `else` branch? I think it's better to specify them explicitly.

Reply:
I Added new cases, but I'm not sure i handled them all

Reply:
Then I suggest to return `null` from `else` and also log, which node type was given, like it's done in similar case in another rule, see `WhiteSpaceRule.handleColon`.

Reply:
Actually, it's not a good idea, because we will spam a lot in logs, let's keep simple `return null`.